mods/ctf/ctf_colors/hud.lua

```diff
@@ -6,14 +6,14 @@ function ctf_colors.get_color(tplayer)
 	end
 	local tcolor_hex = ctf.flag_colors[tcolor_text]
 	if not tcolor_hex then
-		tcolor_hex = "0x000000"
+		tcolor_hex = 0x000000
 	end
 
 	local tcolor_css = "#" .. tcolor_hex:sub(3, 8)
```

Comment:
This line causes a crash, as `tcolor_hex` isn't a string now.

```suggestion
	local tcolor_css = "#" .. tostring(tcolor_hex):sub(3, 8)
```